hpke/hpke.go

```diff
@@ -19,27 +19,30 @@ import (
 
 const versionLabel = "HPKE-06"
 
-// Exporter allows exporting secrets from an HPKE context using a
-// variable-length PRF.
-type Exporter interface {
+// Context defines the capabilities of an HPKE context.
+type Context interface {
 	// Export takes a context string expCtx and a desired length (in bytes),
 	// and produces a secret derived from the internal exporter secret using
 	// the corresponding KDF Expand function. It panics if length is greater
 	// than 255*N bytes, where N is the size (in bytes) of the KDF's output.
 	Export(expCtx []byte, length uint) []byte
+	// Suite returns the cipher suite corresponding to this context.
+	Suite() Suite
+	// Marshal returns the binary representation of the context.
+	Marshal() ([]byte, error)
```

Comment:
I see what you did, so yes you can enforce `binary.MarshalBinary`

Reply:
Done.